'cnd up' crashed due to conversion issues

**Describe the bug**
I had `cnd up` running for a while. At some point, it crashed due to a panic.
```
Waiting for changes to input files of tasks...
panic: interface conversion: runtime.Object is nil, not *v1.Event

goroutine 66 [running]:
github.com/okteto/cnd/pkg/k8/deployments.GetPodEvents(0x222f800, 0xc000be2180, 0xc0011ad880, 0xc000b79200)
	/Users/ramiro/go/src/github.com/okteto/cnd/pkg/k8/deployments/deployments.go:133 +0x5a3
created by github.com/okteto/cnd/cmd.executeUp
	/Users/ramiro/go/src/github.com/okteto/cnd/cmd/up.go:85 +0x5bf
```


**Desktop (please complete the following information):**
 - OS: macOS
 - 0.4.1

--- a/pkg/k8/deployments/deployments.go
+++ b/pkg/k8/deployments/deployments.go
@@ -129,7 +129,17 @@ func GetPodEvents(ctx context.Context, pod *apiv1.Pod, c *kubernetes.Clientset) 
 	for {
 		select {
 		case e := <-ch:
-			event := e.Object.(*apiv1.Event)
+			if e.Object == nil {
+				log.Infof("received an nil event.Object from the pod event call: %+v", e)
+				continue
+			}
+
+			event, ok := e.Object.(*apiv1.Event)
+			if !ok {
+				log.Infof("couldn't convert e.Object to apiv1.Event: %+v", e)
+				continue
+			}
+
 			if event.Type == "Normal" {
 				log.Debug(event.Message)
 			} else {

